lang/src/org/partiql/lang/eval/NaturalExprValueComparators.kt

```diff
@@ -55,9 +55,17 @@ import org.partiql.lang.util.isZero
  *  * `BAG` values come finally (except with [NullOrder.NULLS_LAST]), and their values
  *    compare lexicographically based on the *sorted* child elements.
  */
-enum class NaturalExprValueComparators(private val nullOrder: NullOrder) : Comparator<ExprValue> {
-    NULLS_FIRST(NullOrder.FIRST),
-    NULLS_LAST(NullOrder.LAST);
+enum class NaturalExprValueComparators(private val order: Order, private val nullOrder: NullOrder) : Comparator<ExprValue> {
```

Comment:
nit (documentation): since we're adding another argument to the public constructor, could add a kdoc comment for the new parameter, `order`

Reply:
fixed with new commit